tedana/model/monoexponential.py

```diff
@@ -22,22 +22,22 @@ def fit_decay(data, tes, mask, masksum, start_echo):
     masksum : (S, ) array_like
         Valued array indicating number of echos that have sufficient signal in
         given sample
-    start_echo : int
-        First echo to consider
+    start_echo : int, optional
+        First echo to consider. Default is 1 (first echo).
 
     Returns
     -------
-    t2sa : (S x E) :obj:`numpy.ndarray`
+    t2sa : (S) :obj:`numpy.ndarray`
         Limited T2* map
-    s0va : (S x E) :obj:`numpy.ndarray`
+    s0va : (S) :obj:`numpy.ndarray`
         Limited S0 map
-    t2ss : (S x E) :obj:`numpy.ndarray`
+    t2ss : (S, ?) :obj:`numpy.ndarray`
```

Comment:
Is it unclear what the returned dimensions are, here ?

Reply:
Yes, I'd noticed that too when going through the code a while back. You'll notice that actually overwrites itself; since it's `range(start_echo, n_echos + 1)` when it gets to n_echos it will index the same location. Completely bonkers and I just didn't have the headspace to figure out *why* it was happening...

Reply:
You're right, and I think that's incorrect behavior. Going back to the [original code](https://github.com/ME-ICA/me-ica/blob/master/meica.libs/t2smap.py#L235), zero should be the first value. Nice catch !

Reply:
I think the `start_echo` kwarg was added in the first place to address this discrepancy (but you did that, @emdupre, so you would know better than any of us!). That being said, it might be worth digging into *why* they were ever different. For now we should retain this nonsense functionality so that `tedana.workflows.tedana()` is the same. Indeed, `tedana.workflows.t2smap()` [handles the discrepancy correctly](https://github.com/ME-ICA/tedana/blob/master/tedana/workflows/t2smap.py#L48).

Reply:
Yes, I'm remembering (slowly) that I added the kwarg there to try and align them. @rmarkello, are you saying we should keep the bug ? I guess this is something we could address in #25 ... we should make a note on that issue, if so.

Reply:
No, no, we shouldn't keep it, you're right. But I do think it should potentially be addressed in #25 / a separate PR, just to retain some semblance of order as we try and make PRs more feature-based and less gigantic overhauls of the code.

To @KirstieJane: thank you! Your continued support for all things `tedana` is :100: :blush: :+1:

Reply:
I think that's a reasonable course of action, @rmarkello ! @tsalo, would you mind adding that issue to #25 so we maintain a record ?

Thanks everyone 💖